### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by jim-rosales** [Raggstar/bwui-capstone-placeholder#7]
Grade: Meets Expectations

Summary:
Paul, congrats on finishing your capstone project! Overall you did an excellent job making both your CSS and HTML files easy to read considering you commented both thoroughly. Visually I did like the color scheme you chose and the box-shadows you added were a nice visual addition.

One thing to keep in mind as you work on future projects is syntactical errors as I saw a couple in your CSS file. Browsers are very lenient when it comes to syntax errors as they will try their best to fix these but it is best to avoid this entirely and omit mistakes entirely.

As extra practice and a challenge adding some [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) to your project to make it a bit more [responsive](https://en.wikipedia.org/wiki/Responsive_web_design).

**Comment by Raggstar** [comment]
Jim, many thanks for the review - I much appreciate you comments.  I edited, re-edited.....this was my first web project and it took a bit to realize there were 3 style sheets attached and that each played a specific part in the project.  
I did leave a lot of edit marks that in review I should have deleted.  I have sense installed a program in my editor to help with completing/writing code.  
Many thanks,
Paul Coffey
